pkgs/os-specific/linux/libsemanage/default.nix

```diff
@@ -13,7 +13,13 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ bison flex ];
   buildInputs = [ libsepol libselinux ustr bzip2 libaudit ];
 
-  NIX_CFLAGS_COMPILE = "-fstack-protector-all -std=gnu89";
+  NIX_CFLAGS_COMPILE = [
+    "-fstack-protector-all"
```

Comment:
can’t we remove this? IIRC the stackprotector is now enabled by default since #12895 ?